tests/requirements/python-requirements.txt

```diff
@@ -10,7 +9,6 @@ paramiko==2.12.0
 py3dns==3.2.1
 pymongo==4.3.3
 PyNaCl==1.5.0
-pyrsistent==0.19.3
```

Comment:
Why is this not needed as well? (just for the understanding)

Reply:
It's one of the dependencies of that docker-compose version, which is no longer needed (I verified it in a local Docker container).

Reply:
Thanks @kacf :+1: